frontera/contrib/scrapy/converters.py

```diff
@@ -26,7 +28,7 @@ def to_frontier(self, scrapy_request):
         if callable(eb):
             eb = _find_method(self.spider, eb)
 
-        scrapy_meta = scrapy_request.meta
+        scrapy_meta = dict_to_bytes(scrapy_request.meta, keys_only=True)
```

Comment:
The answer to your question is in the code. Here it is written, that without ``use_bin_type=True`` the string content is always decoded to bytes (or str in Py2). https://pypi.python.org/pypi/msgpack-python
BTW @Preetwinder msgpack-python provides out of the box support of string type encoding. Probably we should check closer, and may be relax rules for meta content.